spec/lib/clarivate/links_client_spec.rb

```diff
@@ -98,4 +98,21 @@
       expect(request_xml).to include fields.first
     end
   end
+
+  describe '.working?' do
+    context 'success' do
+      it 'returns true when it works (has 2 results)' do
+        response_xml = File.read('spec/fixtures/clarivate/links_response.xml')
+        allow(links_client.send(:connection)).to receive(:post).with(any_args).and_return(double(body: response_xml))
+        expect(described_class.working?).to be true
+      end
+    end
```

Comment:
Try 
```ruby
context 'success', :vcr do
  it 'returns true when it works (has 2 results)' do
    expect(described_class.working?).to be true
  end
end
```
The VCR cassette should have the results expected.  This spec might benefit from using VCR.